package.json

```
@@ -19,13 +19,14 @@
     "format": "prettier --write .",
     "clean": "rimraf dist",
     "lint": "eslint src/js",
+    "premanifest": "cp -R src/js/ dist/",
+    "manifest": "npx @custom-elements-manifest/analyzer analyze --config scripts/custom-elements-manifest.config.js",
```

we should be able to use `cem` or `custom-elements-manifest` as the bin name instead of going through npx here.
```suggestion
    "manifest": "cem analyze --config scripts/custom-elements-manifest.config.js",
```

---

this should be possible now since we don't run `yarn manifest` from the docs root. 
I'll follow up on this one.